libs/util/system.h

```diff
@@ -238,6 +239,20 @@ betoh (T const& x)
 #   error "Couldn't determine host endianess!"
 #endif
 
+template <typename T>
```

Comment:
Please include istream.
Please format the functions according to the style of the rest of the functions, i.e., indentation and bracket position.
Please also add the function signature above (below the betoh signature). These files are generally organized to have signatures on top, and implementation here.

Reply:
Ouch, how could I miss that.